x11-misc/sddm/sddm-0.11.0-r2.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit cmake-utils toolchain-funcs user
+
+DESCRIPTION="Simple Desktop Display Manager"
+HOMEPAGE="https://github.com/sddm/sddm"
+SRC_URI="http://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+# KEYWORDS="~amd64 ~arm ~x86"
```

Comment:
It's nicer (IMO) to set it explicitly `KEYWORDS=""`... you know, just in case, we don't want any surprises.